server/status.go

```diff
@@ -92,6 +92,10 @@ const (
 	// healthEndpoint is a shortcut for local details, intended for use by
 	// monitoring processes to verify that the server is up.
 	healthEndpoint = "/health"
+
+	// prometheusEndpoint is the prometheus metrics endpoint, not to be
+	// confused with the built-in cockroach metrics.
+	prometheusEndpoint = "/metrics"
```

Comment:
The default location prometheus expects is `/metrics`.  It's fine to change but if there's no good reason to, it should stay on `/metrics`

There are other metrics collection systems that also use the "Prometheus" format.  It's becoming a defacto standard.